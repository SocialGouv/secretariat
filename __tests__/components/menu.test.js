import { render } from "@testing-library/react"

import Menu from "../../src/components/menu"

jest.mock("next/router", () => ({
  useRouter() {
    return {
      route: "/",
      pathname: "",
      query: "",
      asPath: "",
    }
  },
}))

it("renders homepage", () => {
  const { container } = render(<Menu />)
  expect(container).toMatchSnapshot()
})
