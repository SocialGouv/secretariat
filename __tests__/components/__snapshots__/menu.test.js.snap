// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders homepage 1`] = `
<div>
  <ul
    class="menu"
  >
    <li
      class="selected"
    >
      <a
        href="/"
      >
        Users 
        <span>
          (
          0
          )
        </span>
      </a>
    </li>
    <li
      class=""
    >
      <a
        href="/service/github"
      >
        github
         
        <span>
          (
          0
          )
        </span>
      </a>
    </li>
    <li
      class=""
    >
      <a
        href="/service/matomo"
      >
        matomo
         
        <span>
          (
          0
          )
        </span>
      </a>
    </li>
    <li
      class=""
    >
      <a
        href="/service/mattermost"
      >
        mattermost
         
        <span>
          (
          0
          )
        </span>
      </a>
    </li>
    <li
      class=""
    >
      <a
        href="/service/nextcloud"
      >
        nextcloud
         
        <span>
          (
          0
          )
        </span>
      </a>
    </li>
    <li
      class=""
    >
      <a
        href="/service/ovh"
      >
        ovh
         
        <span>
          (
          0
          )
        </span>
      </a>
    </li>
    <li
      class=""
    >
      <a
        href="/service/sentry"
      >
        sentry
         
        <span>
          (
          0
          )
        </span>
      </a>
    </li>
    <li
      class=""
    >
      <a
        href="/service/zammad"
      >
        zammad
         
        <span>
          (
          0
          )
        </span>
      </a>
    </li>
  </ul>
</div>
`;
